src/Sarif.Driver.UnitTests/TestRuleBase.cs

```diff
@@ -54,17 +54,14 @@ public override RuleConfiguration Configuration
 
         public override Message Help { get { return new Message() { Text = "[Empty]" }; } }
 
-        public abstract void Analyze(TestAnalysisContext context);
```

Comment:
>Analyze [](start = 29, length = 7)

Yes, now that we are an actual `Skimmer` instead of an `ISkimmer`, we already _have_ an abstract `Analyze` method. And that's why everything else is now an override. Nice.